plugins/Wallet/js/reducers/newwalletdialog.js

```diff
@@ -0,0 +1,20 @@
+import { Map } from 'immutable';
+import { SHOW_NEW_WALLET_DIALOG, DISMISS_NEW_WALLET_DIALOG } from '../constants/wallet.js';
+
+const initialState = Map({
+	password: '',
+	seed: '',
+});
+
+export default function newwalletdialog(state = initialState, action) {
+	switch (action.type) {
+	case SHOW_NEW_WALLET_DIALOG:
+		return state.set('password', action.password)
+                .set('seed', action.seed)
+								
+	case DISMISS_NEW_WALLET_DIALOG:
+		return state.set('visible', false);
```

Comment:
looks like this was left over. Double-check for any other overlooked `visible` vars.